src/responder/pam/pamsrv_passkey.c

```diff
@@ -489,6 +489,20 @@ errno_t process_passkey_data(TALLOC_CTX *mem_ctx,
         goto done;
     }
 
+    /* Don't try to process smartcard of sshpubkey mapping data */
+    ret = split_on_separator(tmp_ctx, (const char *) el->values[0].data, ':', true, true,
+                             &mappings, NULL);
+    if (ret != EOK) {
+        DEBUG(SSSDBG_OP_FAILURE, "Incorrectly formatted passkey data [%d]: %s\n",
+                                 ret, sss_strerror(ret));
```

Comment:
Fix indentation, or at least keep it consistent for all `DEBUG` sentences.

Reply:
sorry I don't see the indentation issue, can you clarify?

Reply:
Fixed, please check.